Published weekly changelog (#382)
CoboGlobal/product-manual: en/portal/changelog.mdx

description: "Stay updated with the latest documentation changes for Cobo Portal."
---

<Update label="2025/01/15 - 2025/01/21">
**Organization Management**

> - Enhanced [Assign user roles to members](/en/portal/organization/assign-roles-to-members) with detailed instructions for viewing role details and setting wallet scope

Reviewer: ```suggestion
- Enhanced [Assign user roles to members](/en/portal/organization/assign-roles-to-members) with detailed instructions for setting user roles and wallet scope
```

> - Added new screenshots for member invitation and role modification dialogs

Reviewer: ```suggestion
```

> - Updated documentation to clarify that wallet scope cannot be modified for Admin, Manager, and Approver roles

Reviewer: ```suggestion
```

**Developer Console**
- Enhanced [Register an API key](/en/portal/developer-console/create-api-key) documentation with detailed steps for configuring user roles and wallet scope
- Added prerequisites and improved organization in [Register a webhook endpoint](/en/portal/developer-console/webhooks-create) and [Register a callback endpoint](/en/portal/developer-console/callbacks-register)

**Fee Station**
- Updated [Overview](/en/portal/fee-station/overview) to reflect changes in MPC transaction fee settlement options

> - Removed USDT payment option for MPC transaction fees

Reviewer 2: ```suggestion
```

**MPC Wallets**
- Updated TSS Node version requirement to 0.10.0 in [Verify key shares](/en/portal/mpc-wallets/ocw/key-share-verification)

**Token Bridge**
- Updated [Introduction to Token Bridge](/en/apps/bridge/introduction) to remove Token Conversion feature

**Batch Payouts**

> - Updated [Introduction to Batch Payouts](/en/apps/batch-payouts/introduction), [Make a batch payout](/en/apps/batch-payouts/make-payout), and [Manage batch payouts](/en/apps/batch-payouts/manage-payouts) to clarify that individual transaction management is not supported

Reviewer: ```suggestion
- Updated the information about recipients confirmation on Cobo Guard in [Introduction to Batch Payouts](/en/apps/batch-payouts/introduction) and [Make a batch payout](/en/apps/batch-payouts/make-payout).
```

> - Added information about Cobo Guard confirmation process

Reviewer: ```suggestion
```
